src/memory/redis.rs

```diff
@@ -28,10 +28,10 @@ return value
 
 #[derive(Clone)]
 pub struct RedisMemory<Address, Word> {
-    manager: ConnectionManager,
-    script_hash: String,
-    a: PhantomData<Address>,
-    w: PhantomData<Word>,
+    pub manager: ConnectionManager, // is Send + Sync : https://docs.rs/redis/latest/redis/aio/struct.ConnectionManager.html#impl-Send-for-ConnectionManager
+    script_hash: String,            // trivially Send + Sync
+    a: PhantomData<Address>, // is send + Sync because the underlying address we will use in findex is Send + Sync ([u8; ADDRESS_LENGTH])
+    w: PhantomData<Word>,    // same as above
 }
```

Comment:
about this code : 
```rust
#[derive(Clone)]
pub struct RedisMemory<Address, Word> {
    pub manager: ConnectionManager, // is Send + Sync : https://docs.rs/redis/latest/redis/aio/struct.ConnectionManager.html#impl-Send-for-ConnectionManager
    script_hash: String,            // trivially Send + Sync
    a: PhantomData<Address>, // is send + Sync because the underlying address we will use in findex is Send + Sync ([u8; ADDRESS_LENGTH])
    w: PhantomData<Word>,    // same as above
}
```